lib/fluent/plugin/kubernetes_metadata_common.rb

```diff
@@ -0,0 +1,85 @@
+#
+# Fluentd Kubernetes Metadata Filter Plugin - Enrich Fluentd events with
+# Kubernetes metadata
+#
+# Copyright 2015 Red Hat, Inc.
```

Comment:
This needs to be 2017 - we need to fix the linter or whatever is preventing us from using the correct date

Reply:
We may wish to do this in a subsequent PR since I believe it will snag all files that do not match unless there is a way to ignore year